app/experimenter/templates/experiments/intent_to_ship.txt

```diff
@@ -0,0 +1,26 @@
+{% autoescape off %}
```

Comment:
👍 I like putting the template in a text file like this, we should probably adopt this pattern for the other emails we send.

Reply:
Full credit to @peterbe for the initial implementation, which did this!